docker-compose.yml

```diff
@@ -167,14 +167,16 @@ services:
       ORCHESTRATOR_SENDER_TRANSPORT_USERNAME: admin
       ORCHESTRATOR_SENDER_TRANSPORT_PASSWORD: admin
       SECRETS_PROVIDER_NAME: fileBasedSecretsProvider
-      FILE_BASED_PATH: "/etc/fileBasedSecretStorage/secrets.storage"
+      FILE_BASED_PATH: "/etc/fileBasedSecretStorage/.env.development"
       GRAPHITE_HOST: "graphite"
       GRAPHITE_PORT: 2003
       GRAPHITE_PROTOCOL: PLAINTEXT
+      CORE_SECRET_PROVIDER: secret-file
+      CORE_SECRET_FILES: /etc/fileBasedSecretStorage/.env.development
     volumes:
       - type: bind
-        source: ~/.ort/server/secrets
-        target: /etc/fileBasedSecretStorage
+        source: ./.env.development
+        target: /etc/fileBasedSecretStorage/.env.development
```

Comment:
I'm a little confused here why this is required, the database and RabbitMQ credentials should already be configured via environment variables?

Reply:
There are indeed some inconsistencies in the way secrets are handled, I will merge this as is if it fixes the issue for now and investigate separately what is the best way forward.